en/docs/develop/test/code-coverage.md

```
@@ -1,34 +1,77 @@
+## Generate coverage reports
+
+Add the `--code-coverage` flag along with `--test-report` to generate an HTML report that includes coverage data.
+
+```bash
+# HTML test report with code coverage
+bal test --test-report --code-coverage
 ```
 
 Coverage results are generated at:
+
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)